### ISSUE TITLE
Issue install r-mixomics on mac

### ISSUE THREAD

**Issue by mortonjt** [bioconda/bioconda-recipes#3341]
Hello.  I'm having a bit of trouble installing `r-mixomics` through the bioconda channel.

Specifically, I try to boot up a new environment
```
conda create -n mixomics -c r r ipython-notebook r-irkernel jupyter r-ggplot2 r-dplyr r-essentials python=3.5 numpy scipy pandas matplotlib seaborn hdf5 h5py scikit-bio r-igraph r-rgl
```
And then attempting to install `r-mixomics` via
```
source activate mixomics
conda install -c bioconda r-mixomics=6.1.1
```

However, whenever I run this command, I get the following error
```
Fetching package metadata ...........
Solving package specifications: .


PackageNotFoundError: Package not found: '' Package missing in current osx-64 channels: 
  - r-mixomics 6.1.1*

You can search for packages on anaconda.org with

    anaconda search -t conda r-mixomics
```

Not sure what is going on - it looks like there is an osx version on the bioconda channel.  
Can someone confirm that this is working?  Thanks!

**Comment by bgruening** [comment]
Sorry there is no OSX package for the 6.1.1 version of this package.
https://anaconda.org/bioconda/r-mixomics/files

Maybe the bioconda OSX team can help you? @bioconda/osx

**Comment by bgruening** [comment]
@mortonjt I looked a little bit into it and the problem is that this package depends on libglu. libglu is only available for Linux, I was not able at that time to get it running under OSX. So the correct approach would be to get libglu compiled under OSX.
https://github.com/conda-forge/libglu-feedstock

**Comment by mortonjt** [comment]
Awesome!  Thanks for looking into it!

**Comment by acaprez** [comment]
`r-mixomics` is available now for OS X (PR #3620).  Please note it requires XQuartz to be installed.

**Comment by mortonjt** [comment]
Woo!  Thanks!
